crowsnest/v4l2/raw.py

```
@@ -0,0 +1,195 @@
+    class ValueUnion(ctypes.Union):
+        _fields_ = [
+            ("value", ctypes.c_int32),
+            ("value64", ctypes.c_int64),
+            ("string", ctypes.POINTER(ctypes.c_char)),
+            ("p_u8", ctypes.POINTER(ctypes.c_uint8)),
+            ("p_u16", ctypes.POINTER(ctypes.c_uint16)),
+            ("p_u32", ctypes.POINTER(ctypes.c_uint32)),
+            ("p_s32", ctypes.POINTER(ctypes.c_int32)),
+            ("p_s64", ctypes.POINTER(ctypes.c_int64)),
+            ("ptr", ctypes.POINTER(None)),
+        ]
```

Ruff (0.15.1)

[warning] Mutable default value for class attribute

(RUF012)